src/gc/gc.cpp

```diff
@@ -34212,7 +34212,7 @@ HRESULT GCHeap::Initialize()
 
 #ifdef MULTIPLE_HEAPS
     nhp_from_config = static_cast<uint32_t>(GCConfig::GetHeapCount());
-    
+
```

Comment:
A nit - can you please revert the change in this file as it seems to be an accidental remainder of removed changes?